server/app/models/CategoryModel.java

```diff
@@ -0,0 +1,80 @@
+package models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.ebean.annotation.DbJsonB;
+import io.ebean.annotation.WhenCreated;
+import io.ebean.annotation.WhenModified;
+import java.time.Instant;
+import java.util.List;
+import java.util.Locale;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import play.data.validation.Constraints;
+import services.LocalizedStrings;
+
+/**
+ * An EBean mapped entity representing a category of programs. For example, "Health" or "Education".
+ *
+ * <p>A program can have many categories and a category can be assigned to many programs. Each
+ * category contains a {@code LocalizedStrings} object with the translations of its name.
+ */
```

Comment:
**thought:** We may consider adding an adminName field in the future once these categories are editable by admins and / or if there's any categories exposed in the API, but seems fine to exclude for now

Reply:
Sounds good.  I responded in the TDD so that we can continue exploring that idea there.